src/components/molecules/UserList/UserList.tsx

```diff
@@ -38,7 +36,8 @@ export const UserList: React.FC<UserListProps> = ({
   showTitle = true,
   attendeesTitle,
 }) => {
-  const hasExcessiveUserCount = userCount > usersSample.length;
+  const excessiveUserCount =
+    userCount - usersSample.length > 0 ? userCount - usersSample.length : 0;
```

Comment:
In case of `NaN` (is that possible?), fallback to empty string ?

I think the code from bellow `and {excessiveUserCount} more` would actually read `and more` in case of NaN

Reply:
since we have this true/false check in the render that shouldn't be the case 
`{excessiveUserCount && (
          <div className="UserList__excessive-number">
            and {excessiveUserCount} more
          </div>
        )}
`